mappings/net/minecraft/client/render/block/entity/HangingSignBlockEntityRenderer.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_7761 net/minecraft/client/render/block/entity/HangingSignBlockEntityRenderer
+	FIELD field_40516 NORMAL_CHAINS Ljava/lang/String;
+	FIELD field_40517 CHAIN_L1 Ljava/lang/String;
+	FIELD field_40518 CHAIN_L2 Ljava/lang/String;
+	FIELD field_40519 CHAIN_R1 Ljava/lang/String;
+	FIELD field_40520 CHAIN_R2 Ljava/lang/String;
+	FIELD field_40521 BOARD Ljava/lang/String;
+	FIELD field_40522 PLANK Ljava/lang/String;
+	FIELD field_40523 V_CHAINS Ljava/lang/String;
+	FIELD field_40524 MODELS Ljava/util/Map;
+	METHOD method_45791 getRotationDegrees (Lnet/minecraft/class_2680;Z)F
+		ARG 1 state
+		ARG 2 wall
+	METHOD method_45794 (Lnet/minecraft/class_5614$class_5615;Lnet/minecraft/class_4719;)Lnet/minecraft/class_7761$class_7762;
+		ARG 1 type
+	METHOD method_45795 getHangingTexturedModelData ()Lnet/minecraft/class_5607;
```

Comment:
```suggestion
	METHOD method_45795 getTexturedModelData ()Lnet/minecraft/class_5607;
```
enigma doesnt like things having the same name as parent classes even though java's fine with it